app/controllers/passengers_controller.rb

```diff
@@ -102,13 +89,28 @@ def disallow_nonexpiring_note(permitted_params)
     permitted_params.except :doctors_note_attributes
   end
 
+  def find_passenger
+    @passenger = Passenger.find(params[:id])
+  end
+
+  def passenger_params
+    base_passenger_params
+      .then { |p| restrict_admin p }
+      .then { |p| disallow_nonexpiring_note p }
+  end
+
+  def passenger_pdf
+    @passengers = @passengers.send(@filter) if @filter.present?
+    pdf = PassengersPDF.new(@passengers, @filter)
+    name = "#{@filter} Passengers #{Date.today}".capitalize
```

Comment:
So if filter isn't present the name will just be " Passengers etc"? Is it purposeful that you're doing away with "All Passengers" as a possible title?
A possible elegant solution if you did want to keep that at all would be to have `@filter ||= 'all'` in the beginning of the method.